src/status_im2/subs/contact.cljs

```diff
@@ -228,15 +226,18 @@
  (fn [[contact] _]
    (:blocked contact)))
 
+;; TODO(alwx):
```

Comment:
could you add a description here or link an issue if you want do it in a separate PR?